### ISSUE TITLE
Tags list below the input...

### ISSUE THREAD

**Issue by juddey** [maxmarcon/live_select#25]
Hi Max

Thanks for this great lib. I've got a requirement to move the tags list underneath the wrapper - so it ends up looking something like this:

![image](https://user-images.githubusercontent.com/7445602/235016623-3776ae9a-917e-46ee-9066-7a59801d93a4.png)

Would you be interested in a PR for this functionality? If not, I'll just keep it to my own fork.

Cheers :)

**Comment by maxmarcon** [comment]
Hi @juddey and thanks!

Are you using tailwind? I believe that what you want can be acheived using the styling options. So with tailwind, you would do this:

﻿﻿https://live-select.fly.dev/?active_option_class=&available_option_class=&container_class=&container_extra_class=flex+flex-col&debounce=&disabled=false&dropdown_class=&dropdown_extra_class=top-11&max_selectable=0&mode=tags&option_class=&option_extra_class=&placeholder=Search+for+a+city&search_delay=10&selected_option_class=&show_styles=true&style=tailwind&style_filter=&tag_class=&tag_extra_class=&tags_container_class=&tags_container_extra_class=order-last&text_input_class=&text_input_extra_class=&text_input_selected_class=&update_min_len=3&user_defined_options=false

Basically, making the component outer container a flex column, forcing the tags container to be the last element, and moving down the absolutely positioned dropdown a bit so that it doesn't overlap the text input.

Let me know.

**Comment by juddey** [comment]
Hey @maxmarcon Thanks, that works well!

I'll close this issue out now.
